src/View/ViewVarsTrait.php

```diff
@@ -26,6 +26,22 @@
 {
 
     /**
+     * The name of default View class.
+     *
+     * @var string
+     */
+    public $viewClass = null;
+
+    /**
+     * View instance.
+     *
+     * Won't be set until after ViewVarsTrait::createView() is called.
+     *
+     * @var \Cake\View\View
+     */
+    public $_view;
```

Comment:
Can we not have an additional property? Would having getView() return new instances totally break things?